azure/appservice-windows/main.tf

```diff
@@ -0,0 +1,89 @@
+resource "azurerm_resource_group" "app" {
+  name     = "rg-${var.project}-${var.environment}"
+  location = local.location
+
+  tags = {
+    country     = var.country
+    environment = var.environment
+    squad       = var.squad
+    product     = var.product
+  }
+}
+
+resource "azurerm_app_service_plan" "app" {
+  name                = "asp-${var.project}-${var.environment}"
+  location            = azurerm_resource_group.app.location
+  resource_group_name = azurerm_resource_group.app.name
+  kind                = "Windows"
+  reserved            = false
+
+  sku {
+    tier = var.plan
+    size = var.size
+  }
+
+  tags = {
+    country     = var.country
+    environment = var.environment
+    squad       = var.squad
+    product     = var.product
+  }
+}
+
+resource "azurerm_app_service" "app" {
+  name                    = "as-${var.project}-${var.environment}"
+  location                = azurerm_resource_group.app.location
+  resource_group_name     = azurerm_resource_group.app.name
+  app_service_plan_id     = azurerm_app_service_plan.app.id
+  https_only              = true
+  enable_client_affinity  = true
+  stack                   = var.stack
+  identity {
+    type = "SystemAssigned"
+  }
+  site_config = {
```

Comment:
We can also add http version 2.0. I believe it's retro compatible and it covers the future